helm/draughtsman-chart/values.yaml

```diff
@@ -0,0 +1,35 @@
+image:
+  registry: "quay.io"
+  repository: "giantswarm/draughtsman"
+  tag: "[[ .SHA ]]"
+pod:
+  user:
```

Comment:
I see this added but missing where it is used

Reply:
https://github.com/giantswarm/draughtsman/pull/86/files#diff-b187610906214a662aeb18f2f7f8de5dR30

I feel that this PR would be better if I dropped the changes to `values.yaml` and add those (and the manifest templating) in a separate PR.